### PR TITLE
code cleanup for kubectl taint

### PR DESCRIPTION
**Pull request by lojies**
**What type of PR is this?**
 /kind cleanup

**What this PR does / why we need it**:
code cleanup for kubectl taint
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
NONE
```release-note

```

### CODE REVIEW

File: pkg/kubectl/cmd/taint/taint.go [hunk 1]
```diff
@@ -98,15 +98,9 @@ func NewCmdTaint(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobra.
 		Long:                  fmt.Sprintf(taintLong, validation.DNS1123SubdomainMaxLength, validation.LabelValueMaxLength),
 		Example:               taintExample,
 		Run: func(cmd *cobra.Command, args []string) {
-			if err := options.Complete(f, cmd, args); err != nil {
-				cmdutil.CheckErr(err)
-			}
-			if err := options.Validate(); err != nil {
-				cmdutil.CheckErr(cmdutil.UsageErrorf(cmd, err.Error()))
```
Comment (justinsb) [inline pkg/kubectl/cmd/taint/taint.go]: I think we've lost the `UsageErrorf` message here.

**Comment by justinsb** [comment]
Looks like a small behavioural change around `Validate`, but other methods do follow this pattern, so I'll leave that decision to the approver.

/lgtm

File: pkg/kubectl/cmd/taint/taint.go (hunk 1 above)
Comment (juanvallejo) [reply pkg/kubectl/cmd/taint/taint.go]: I think that's fine. This follows the pattern seen in other commands: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/set/set_selector.go#L104

**Comment by juanvallejo** [comment]
cc @soltysh for approval

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**